packages/fxa-settings/src/pages/Signup/Confirm/index.tsx

```diff
@@ -2,28 +2,61 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { useState } from 'react';
-import { logViewEvent, usePageViewEvent } from '../../../lib/metrics';
-import { RouteComponentProps } from '@reach/router';
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+  logErrorEvent,
+  logViewEvent,
+  usePageViewEvent,
+} from '../../../lib/metrics';
+import { RouteComponentProps, useNavigate } from '@reach/router';
 import ConfirmWithLink, {
   ConfirmWithLinkPageStrings,
 } from '../../../components/ConfirmWithLink';
-import { REACT_ENTRYPOINT } from '../../../constants';
+import {
+  NAVIGATION_TIMEOUT_MS,
+  POLLING_INTERVAL_MS,
+  REACT_ENTRYPOINT,
+} from '../../../constants';
 import { ResendStatus } from '../../../lib/types';
+import AppLayout from 'fxa-settings/src/components/AppLayout';
+import { useAccount, useInterval } from 'fxa-settings/src/models';
+import { AuthUiErrors } from 'fxa-settings/src/lib/auth-errors/auth-errors';
+import LoadingSpinner from 'fxa-react/components/LoadingSpinner';
 
-export type ConfirmProps = {
-  email: string;
-};
+// This page is no longer part of the typical/expected signup flow, but has been preserverd during
+// the conversion from backbone to react as we were still seeing some traffic to this route.
+// TODO: Check metrics once this React page is rolled out to determine if this page can be entirely replaced by ConfirmSignupCode.
```

Comment:
Do we have a ticket filed for this? If so, we might want to include the Jira ticket number here.